### PR TITLE
PPG with Id for VM/VMSS

### PR DESCRIPTION
**Pull request by sergey-shandar**
## Checklist

- [ ] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [ ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by cormacpayne** [comment]
Sign job: https://azuresdkci.westus2.cloudapp.azure.com/job/psnetsign-esrp/395/

File: sergey.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+```
+VM -> AvailabilitySet -> ProximityPlacementGroup
+   ->                    ProximityPlacementGroup
+```
+
+## Scenarios
+
+- availabilitySet, proximityPlacementGroup
+  - check match.
```
Comment (markcowl) [inline sergey.md]: Can we remove this file?

File: sergey.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Comments
+
```
Comment (markcowl) [inline sergey.md]: Also, still need:
- Update to Compute Changelog.md, 'Added PPG support to New-AzVM and New-AzVmss simple parameter sets'
- Markdown help updates (the syntax diagram has changed slightly
- Regression test (if time)

**Comment by sergey-shandar** [comment]
@markcowl it's ready for review.

File: src/Compute/Compute/help/New-AzVmss.md [hunk 3]
```diff
@@ -392,6 +392,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -ProximityPlacementGroup
+{{ Fill ProximityPlacementGroup Description }}
```
Comment (markcowl) [inline src/Compute/Compute/help/New-AzVmss.md]: Need to fill in the parameter description here, like 
```
The name or resource id of the Proximity Placement Group to use for this Virtual Machine ScaleSet

File: src/Compute/Compute/help/New-AzVM.md [hunk 4]
```diff
@@ -440,6 +440,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -ProximityPlacementGroup
+{{ Fill ProximityPlacementGroup Description }}
```
Comment (markcowl) [inline src/Compute/Compute/help/New-AzVM.md]: Need to fill in the description here, like The name or resource Id of the Proximity Placement Group to use with this VM.

**Pull request merged**